includes/languages/lang.english.php

```diff
@@ -187,6 +187,7 @@
     'ERROR_INVALID_KEYWORDS' => 'Invalid keywords.',
     'ERROR_INVALID_TO_DATE' => 'Invalid To Date.',
     'ERROR_MAXIMUM_QTY' => 'The quantity added to your cart has been adjusted because of a restriction on maximum you are allowed. See this item:<br>',
+    'ERROR_MISSING_SORT_OPTIONS' => 'Missing sort options',
```

Comment:
Should 'sort' be 'search' here? Both in the constant name and its defined value?

Reply:
Whoops, you're right. Will fix.